boms/cloud-oss-bom/pom.xml

```diff
@@ -29,7 +29,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <guava.version>26.0-android</guava.version>
+    <guava.version>27.1-android</guava.version>
```

Comment:
Now that this property is used only once, can we inline it instead of using a property?

Reply:
We can. Personally I like having all the versions in one place though. Anyone else have a strong opinion about this?

Reply:
I thought that single place could be the `<dependencyManagement>` section.

Reply:
We don't have BOMs for everything yet, and even when we do they're still going to be separated form each other by multiple lines.

Reply:
This is up to you all.